examples/SpringHibernateBalancedReaderOneDataSourceExample/README.md

```diff
@@ -68,9 +68,9 @@ Configure Spring to use the AWS JDBC Driver as the default datasource.
           driver-class-name: software.amazon.jdbc.Driver
           type: org.springframework.jdbc.datasource.SimpleDriverDataSource
     ```
-2. The datasource mentioned above does not use Hikari datasource that is default for Spring 2+ application. The AWS JDBC Driver manages its own internal connection pool (or several connection pools, if needed), which increases overall efficiency and helps facilitate failover support. All necessary configuration parameters are defined in the `F0` configuration profile. Other configuration presets `D`, `E` and `F` can be used as well. Any configuration profile or preset specified should use the [Read/Write Splitting Plugin](../docs/using-the-jdbc-driver/using-plugins/UsingTheReadWriteSplittingPlugin.md). More details are available at [Configuration Profiles](../docs/using-the-jdbc-driver/UsingTheJdbcDriver.md#configuration-profiles) and [Configuration Presets](../docs/using-the-jdbc-driver/ConfigurationPresets.md).
+2. The datasource mentioned above does not use Hikari datasource that is default for Spring 2+ application. The AWS JDBC Driver manages its own internal connection pool (or several connection pools, if needed), which increases overall efficiency and helps facilitate failover support. All necessary configuration parameters are defined in the `F0` configuration profile. Other configuration presets `D`, `E` and `F` can be used as well. Any configuration profile or preset specified should use the [Read/Write Splitting Plugin](../../docs/using-the-jdbc-driver/using-plugins/UsingTheReadWriteSplittingPlugin.md). More details are available at [Configuration Profiles](../../docs/using-the-jdbc-driver/UsingTheJdbcDriver.md#configuration-profiles) and [Configuration Presets](../../docs/using-the-jdbc-driver/ConfigurationPresets.md).
    <br><br>
-   Including the optional configuration parameter `readerHostSelectorStrategy` in the connection string helps to set up a strategy to select a reader node. Possible values are `random`, `roundRobin` and `leastConnections`. More details are available at [Reader Selection Strategies](../docs/using-the-jdbc-driver/using-plugins/UsingTheReadWriteSplittingPlugin.md#reader-selection-strategies).
+   Including the optional configuration parameter `readerHostSelectorStrategy` in the connection string helps to set up a strategy to select a reader node. Possible values are `random`, `roundRobin` and `leastConnections`. More details are available at [Reader Selection Strategies](../../docs/using-the-jdbc-driver/using-plugins/UsingTheReadWriteSplittingPlugin.md#reader-selection-strategies).
```

Comment:
I believe the URL should instead be `../../docs/using-the-jdbc-driver/ReaderSelectionStrategies.md`. Unfortunately I think the `#reader-selection-strategies` anchor in `UsingTheReadWriteSplittingPlugin.md` doesn't exist anymore.

Reply:
fixed. thx!